swo/mpt/cli/core/mpt/flows.py

```diff
@@ -1,21 +1,30 @@
+import json
 from typing import Optional
 from urllib.parse import quote_plus
 
-from swo.mpt.cli.core.errors import wrap_http_error
+from swo.mpt.cli.core.errors import MPTAPIError, wrap_http_error
 
 from .client import MPTClient
 from .models import ListResponse, Meta, Product, Token
 
 
 @wrap_http_error
-def get_token(mpt_client: MPTClient, token_id: str) -> Token:
+def get_token(mpt_client: MPTClient, secret: str) -> Token:
     """
     Retrieves API Token from the MPT Platform
     """
-    response = mpt_client.get(f"/accounts/api-tokens/{token_id}")
+    response = mpt_client.get(f"/accounts/api-tokens?limit=2&token={secret}")
```

Comment:
Why limit 2 ?
You could still check the pagination meta to be sure it returned just 1 item in data right ?

Reply:
agree, I'll fix later ;-)